frontend/src/lib/components/Cards/TextCard/TextCard.tsx

```diff
@@ -61,90 +76,100 @@ export function TextCardInternal(
     const otherDashboards = nameSortedDashboards.filter((dashboard) => dashboard.id !== dashboardId)
     return (
         <div
-            className={clsx('TextCard rounded flex flex-col', className, showResizeHandles && 'border')}
+            className={clsx('Card TextCard rounded flex flex-col', className, showResizeHandles && 'border')}
             data-attr="text-card"
             {...divProps}
             ref={ref}
         >
-            <div className={clsx('flex flex-row p-2', showResizeHandles ? 'border-b' : 'border rounded-t')}>
-                <UserActivityIndicator
-                    className={'grow'}
-                    at={text.last_modified_at}
-                    by={text.created_by || text.last_modified_by}
-                />
-                <div className="min-h-4 flex items-center justify-end">
-                    <More
-                        overlay={
-                            <>
-                                <LemonButton
-                                    status="stealth"
-                                    fullWidth
-                                    onClick={() =>
-                                        dashboardId && push(urls.dashboardTextTile(dashboardId, textTile.id))
-                                    }
-                                    data-attr="edit-text"
-                                >
-                                    Edit text
-                                </LemonButton>
+            <CardMeta
+                showEditingControls={showEditingControls}
+                showDetailsControls={true}
+                setAreDetailsShown={setAreDetailsShown}
+                areDetailsShown={areDetailsShown}
+                className={clsx(showResizeHandles ? 'border-b' : 'border rounded-t')}
+                metaDetails={
+                    <UserActivityIndicator
```

Comment:
The feedback about using the "Created by" format in the details part still holds
https://user-images.githubusercontent.com/4550621/199441563-6b5a1d59-526d-4a95-a3ba-2386eaf255fc.png

Reply:
yep, will push this into #12560 wanted to keep this as small as I could get away with